rcl_logging_spdlog/test/fixtures.hpp

```diff
@@ -35,7 +35,7 @@
 #define pclose _pclose
 #define DIR_CMD "dir /B"
 #else
-#define DIR_CMD "ls -d"
+#define DIR_CMD "ls -td"
```

Comment:
actually this was a bug, the original tests were NOT comparing the latest files against the expected result. since this will list the files in time ascending order, i happened to find this when adding the test. this will align the same behavior with windows, to list time descending order.